docs/execution_providers/ACL-ExecutionProvider.md

```diff
@@ -7,19 +7,11 @@ For build instructions, please see the [BUILD page](../../BUILD.md#ARM-Compute-L
 
 ### Using the ACL execution provider
 #### C/C++
-To use ACL as execution provider for inferencing, please register it as below.
 ```
-string log_id = "Foo";
-auto logging_manager = std::make_unique<LoggingManager>
-(std::unique_ptr<ISink>{new CLogSink{}},
-                                  static_cast<Severity>(lm_info.default_warning_level),
-                                  false,
-                                  LoggingManager::InstanceType::Default,
-                                  &log_id)
-Environment::Create(std::move(logging_manager), env)
-InferenceSession session_object{so, env};
-session_object.RegisterExecutionProvider(std::make_unique<::onnxruntime::ACLExecutionProvider>());
-status = session_object.Load(model_file_name);
+Ort::Env env = Ort::Env{ORT_LOGGING_LEVEL_ERROR, "Default"};
```

Comment:
the previous internal C++ api examples create a session.
shouldn't we do the same for the corrected examples?
create a session with the session options that you initialized with the various EP options?